.github/workflows/_test_slurm_pyxis.yaml

```diff
@@ -78,15 +68,21 @@ on:
         value: ${{ jobs.run-test.outputs.SLURM_EXITCODE }}
 
 jobs:
-
   run-test:
     name: ${{ inputs.NAME }}
     runs-on: jumpbox
+    env:
+      OUTPUT_BASEDIR: /nfs/cluster
+      SLURM_CLUSTER: ${{ inputs.GPU_TYPE == 'A100' && secrets.CLUSTER_LOGIN_USER || secrets.AWS_LOGIN_USER }}@${{ inputs.GPU_TYPE == 'A100' && vars.HOSTNAME_SLURM_LOGIN || vars.HOSTNAME_AWS_SLURM }}
```

Comment:
This logic, if necessary, shall be better placed in the calling workflow to maximize reusability of this workflow (which is essentially a function to be called by the master workflow).